src/index.js

```diff
@@ -70,14 +102,36 @@ class ServerlessLeo {
 
     this.hooks = {
       'create:bot:copy-template': () => {
-        this.serverless.pluginManager.cliOptions['template-url'] = 'https://github.com/LeoPlatform/serverless-leo/tree/master/templates/bot'
-        const { name } = this.serverless.pluginManager.cliOptions
-        this.serverless.pluginManager.cliOptions['path'] = `bots${path.sep}${name}`
+        let {
+          language,
+          name,
+          path: outputPath,
+          type
+        } = this.serverless.pluginManager.cliOptions
+        outputPath = outputPath || `bots${path.sep}${name}`
+
+        const templateUrl = `https://github.com/LeoPlatform/serverless-leo/tree/master/templates/bot/${language}/${type}`
+        this.options['template-url'] = templateUrl
+        this.options.path = outputPath
+
+        this.serverless.pluginManager.cliOptions['template-url'] = templateUrl // TODO: old version of serverless?
```

Comment:
I left this in as it was the original code, but changes had to be made to `this.options` instead of `this.serverless.pluginManager.cliOptions`